sdk/samples/recver_app.c

```diff
@@ -16,7 +16,7 @@
 #include "mcm_dp.h"
 
 #define DEFAULT_RECV_IP "127.0.0.1"
-#define DEFAULT_RECV_PORT "9001"
+#define DEFAULT_RECV_PORT "9002"
```

Comment:
Why do you need this change?

Reply:
I want to create 2 application on the same machine. The port numbers cannot be the same so I changed port numbers:
- Receiver uses 9002
- Sender uses 9001

Honestly, I don't need to change it here, this is only default val, I could pass correct port number by parameter, but I wanted default value to be something that works

Reply:
Yeah, please remove this change and use the port number argument to set it to 9002.